source/documentation/tools/quicksight/01-index.md

```diff
@@ -0,0 +1,35 @@
+# QuickSight
```

Comment:
Could this be more descriptive? For example: 'Use QuickSight to visualise your data' or similar. That could give users better understanding of whether this content is relevant to them.